src/app/public/modules/lookup/types/lookup-select-mode.ts

```diff
@@ -0,0 +1,13 @@
+export enum SkyLookupSelectMode {
+
+  /**
+   * Allows for the selection of a single item.
```

Comment:
```suggestion
   * Restricts users to a single selection.
```
Just my 2 cents, but I think phrasing this as `Restricts...` provides a better indication of the use case for this option. But if you think we should keep parallel construction in the descriptions, we can change this to `Allows users to select a single item.`

Reply:
Done